Ginger/GingerCoreCommon/ReportsLib/ReportInfoAttribute.cs

```diff
@@ -73,4 +73,11 @@ public FieldParamsIsSelected(bool IsSelected)
             this.IsSelected = IsSelected;
         }
     }
+
+    public class UsingUTCTimeFormat : Attribute
+    {
+        public UsingUTCTimeFormat()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant constructor.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5538205)